### PR TITLE
Added easy admin crud for funding org

### PR DESCRIPTION
**Pull request by praneethpr**

### CODE REVIEW

File: src/Entity/Entity.php [hunk 1]
```diff
@@ -157,6 +157,22 @@ public function getCreator()
...
+     *
+     * @return string.
+     */
+    public function getCreatorName(): string
+    {
+        $creatorName = '';
+        if ($this->getCreator()) {
+            $creatorName = $this->getCreator()->getFirstName() . ' ' . $this->getCreator()->getLastName();
+        }
+        return $creatorName;
```
Comment (mickel1138) [inline src/Entity/Entity.php:173]:
```suggestion
        return $creatorName ?? '';
```

Comment (praneethpr) [reply src/Entity/Entity.php:173]: I think initializing the variable to a string is better so that we dont get the variable not defined warning

**Review by fightingtexasaggie — DISMISSED** [review]
Ok, since the FC will be taken care of elsewhere, everything else looks solid. I also tested the delete without linking FCs and was able to delete as expected.

File: src/Controller/Admin/DataRepositoryCrudController.php [hunk 2]
```diff
@@ -0,0 +1,36 @@
+<?php
+
+namespace App\Controller\Admin;
+
+use App\Entity\DataRepository;
+use EasyCorp\Bundle\EasyAdminBundle\Controller\AbstractCrudController;
+use EasyCorp\Bundle\EasyAdminBundle\Field\IdField;
+use EasyCorp\Bundle\EasyAdminBundle\Field\TextField;
+
+class DataRepositoryCrudController extends AbstractCrudController
```
Comment (mickel1138) [inline src/Controller/Admin/DataRepositoryCrudController.php:10]: Needed?

**Review by mickel1138 — APPROVED** [review]
LGTM

**Pull request merged**